spec/features/admin/enterprises/business_address_form_spec.rb

```diff
@@ -0,0 +1,143 @@
+require "spec_helper"
+
+feature "Business Address" do
```

Comment:
@filipefurtad0 is currently doing some great work to move feature specs to system specs. In that process it would be easier if we all abandoned the use of `feature` and `scenario`. We should use `describe` and `it` instead.
```suggestion
describe "Business Address" do
```